src/objects-table/ObjectsTable.js

```diff
@@ -130,6 +129,7 @@ class ObjectsTable extends React.Component {
             searchValue: null,
             detailsObject: null,
             selection: new Set(),
```

Comment:
Selection is cleared if a row is clicked and the detail box is open, is that intentional? Also if the contextual menu for a row is open.

Reply:
No, that's a bug!! Thanks :)